src/Container/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/require-default-props */
```

Comment:
[optional] With a recent update to `@edx/eslint-config`, we might be able to upgrade to v4.2.0 in this repo and remove the need for default params in this file without disabling the ESLint rule 👀 https://github.com/openedx/eslint-config/pull/156

While this repo is currently using v3 of `@edx/eslint-config`, the v4 breaking changes don't seem to apply to this repo at the moment, per the release notes, so the upgrade should be fairly trivial: 

> Drops support for `eslint` ^6.8.0; `@edx/eslint-config` must now be used with at least `eslint` versions ^7.32.0 or ^8.2.0.

Reply:
@adamstankiewicz  I tried to update but got the following error on linting (which prevents the commit).
Did you already experienced this error before?

```
> @openedx/paragon@20.18.1 lint
> npm run stylelint && eslint --ext .js --ext .jsx --ext .ts --ext .tsx . && npm run lint --workspaces --if-present


> @openedx/paragon@20.18.1 stylelint
> stylelint "src/**/*.scss" "scss/**/*.scss" "www/src/**/*.scss" --config .stylelintrc.json


Oops! Something went wrong! :(

ESLint: 8.18.0

Error: Error while loading rule '@typescript-eslint/naming-convention': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
Occurred while linting /home/rpenido/Projects/opencraft/paragon/.eslintrc.js
...
```

Reply:
Hmm, I have not. Thanks for flagging. I am able to reproduce the issue on my end as well. Unless you want to investigate the error further, feel free to defer on the upgrade and continue with your existing solution using default props and the disabled ESLint rule. If you do defer on the investigation/resolution, do you mind filing a new Paragon issue to document the error?

Reply:
Issue created: https://github.com/openedx/paragon/issues/3230